app/src/main/java/com/framgia/soundcloud/screen/detail/DetailPresenter.java

```diff
@@ -0,0 +1,32 @@
+package com.framgia.soundcloud.screen.detail;
+
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports